.githooks/pre-commit

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+dotnet format whitespace src/Nethermind/ --folder
```

Comment:
Have you check if this work with windows?

Reply:
Yes i actually use windows machine. you just need to install dotnet format like in the description of the PR